include/posix/lib_posix.h

```diff
@@ -0,0 +1,214 @@
+//  Copyright (c) 2013, Facebook, Inc.  All rights reserved.
```

Comment:
Isn't it confusing if I am adding a header called env_posix.h if I am implementing a new Env?